### PR TITLE
Dynamic CICO Text And Post Exchange Send Flow

### PR DESCRIPTION
**Pull request by MuckT**
### Description

A PR to fix  _Withdraw flow is broken in Add/Withdraw menu_ #224.
- Includes send button from cash out exchange screens.
- Dynamic text based on selected currency in top text on cash out exchange screens.
- Remove Account Address from cash out exchange screens.
- Add Analytics for new flow

### Other changes

- Skipped Bidali cash out test as the custom webview was causing issues with detox.

### Tested

- [x] Tested locally on iOS
- [x] Tested locally on Android
- [x] E2e tests updated

#### Test case - cUSD Cryptocurrency Exchange

1. Go to Settings
2. Tap 'Add & Withdraw'
3. Tap 'Withdraw Funds'
4. Tap 'Celo Dollar (cUSD)' & 'Cryptocurrency exchange'
5. Tap 'Next'
6. Observe top text should say 'You can transfer cUSD to the following exchanges. Once you have retrieved your address for your chosen exchange, you can continue to the next step.
7. Tap 'Send'.
8. Check that you have arrived in the Send Flow and that the confirm your number prompt is not visible.
9. Enter an address - e.g. `0x22c8a9178841ba95a944afd1a1faae517d3f5daa` for Alfajores.
10. Accept send to unknown address prompt if served.
11. `Send cUSD`, cUSD Balance should be visible, and that user is unable to change to cEUR.
12. Complete send flow.

#### Test case - cEUR Cryptocurrency Exchange

1. Go to Settings
2. Tap 'Add & Withdraw'
3. Tap 'Withdraw Funds'
4. Tap 'Celo Euro (cEUR)' & 'Cryptocurrency exchange'
5. If in US, should see 'There are no exchanges available for cEUR in your country.', 'Switch Currency' option & 'Contact Support' option.
6. Check that options work accordingly.

#### Test case - CELO Cryptocurrency Exchange
1. Go to Settings
2. Tap 'Add & Withdraw'
3. Tap 'Withdraw Funds'
4. Tap 'Celo Euro (cEUR)' & 'Cryptocurrency exchange'
5. Should see a list of exchanges and a 'Withdraw CELO' button
6. Tap Withdraw CELO - Should enter send to address flow

### Screenshots
<div>

</div>

### Related issues

- #224
- #921

### Backwards compatibility

This change is backwards compatible

### CODE REVIEW

**Comment by dominiquejane** [comment]
@MuckT Hey, what needs to happen to get this PR approved?

**Comment by MuckT** [comment]
@dominiquejane a review from someone on the engineering team is needed to get this merged. Also, someone from design taking a look and signing off on some of the screens would expedite this process. I'll work on resolving the conflicts so that this can be more effectively reviewed.

**Comment by MuckT** [comment]
Notes from design review with @nityarao:
- Withdraw CELO instead of Send CELO - should match subsequent screen
- Remove QR code form Send Bar cUSD/ cEUR withdrawal
- cUSD/cEUR/CELO address copy change.
- Can we remove the contacts import on if arriving at send flow from withdrawal?
- Can we default to coin in send flow to that selected in path of entry?

File: packages/mobile/src/navigator/types.tsx [hunk 1]
```diff
@@ -200,13 +201,17 @@ export type StackParamList = {
   [Screens.Send]:
     | {
         isOutgoingPaymentRequest?: boolean
+        skipContactsImport?: boolean
+        defaultCurrency?: Currency
```
Comment (gnardini) [inline packages/mobile/src/navigator/types.tsx:205]: Is `defaultCurrency` being used? Otherwise lets remove it :)

**Review by gnardini — APPROVED** [review]
I left two small comments but everything looks really good, great job! 🎉 

One thing we might want is to include analytics for the screen, I saw you left a TODO, but at least just an empty event to have _something_ would be nice.

File: packages/mobile/src/navigator/types.tsx (hunk 1 above)
Comment (MuckT) [reply packages/mobile/src/navigator/types.tsx]: It was not being used. It's removed in https://github.com/valora-inc/wallet/pull/1036/commits/164434e67ea3dca9459b3cd20e1380a81d52c579.

File: packages/mobile/locales/es-419/fiatExchangeFlow.json [hunk 2]
```diff
@@ -61,7 +61,8 @@
   "dailyLimitRequest": "¡Hola! Me gustaría aumentar mi límite de envío diario.",
   "pleaseSelectProvider": "Selecciona un proveedor de la lista a continuación. Estos enlaces te conectarán a servicios proporcionados por terceros.",
   "otherFundingOptions": "Para conocer más formas de agregar y retirar fondos, consulta <0>¿Cómo puedo agregar fondos a mi cuenta de Valora?</0>",
-  "youCanTransfer": "Puedes transferir {{currency}} con los siguientes exchanges de criptomonedas. Necesitarás tu Dirección de cuenta.",
+  "youCanTransferIn": "Puedes transferir {{digitalAsset}} con los siguientes exchanges de criptomonedas. Necesitarás tu Dirección de cuenta.",
```
[the remark this answers is not in the document]
Comment (MuckT) [reply packages/mobile/locales/es-419/fiatExchangeFlow.json]: Resolved in https://github.com/valora-inc/wallet/pull/1036/commits/280ffefa2018ea7c3df7b88b26903e800144c189.

**Comment by MuckT** [comment]
@gnardini added analytics in https://github.com/valora-inc/wallet/pull/1036/commits/a3df9e0899cd1599074fb55d48ceb2e719d1133a!

**Review by gnardini — APPROVED** [review]
🚀

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified this issue using Latest Test flight Build V1.21.0(1635160132) & updated status in Test Quality Tool:
**Test Quality Tool Link:**
**iOS (13.0):** 
- https://valora.testquality.com/project/12128/plan/22175/run/151573/result/2540542

**Verified On Device:** iPhone SE (13.6.1)

**Pull request merged**